### PR TITLE
ci(release): add macos arm64 artifact

### PR DESCRIPTION
**Pull request by killagu**
## Details
Add macos arm64 artifact to github release.

## Types of changes

_What types of changes does your PullRequest introduce? Put an `x` in all the boxes that apply:_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Go over all the following points, and put an `x` in all the boxes that apply._

No test need.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -147,7 +147,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        arch: [amd64]
+        arch: [amd64,arm64]
```
Comment (imeoer) [inline .github/workflows/release.yml:150]: Maybe add space like `[amd64, arm64]`.

**Review by adam3q — COMMENTED** [review]
Can we package the binary in the contrib directory, such as Nydusify, into macos?

**Comment by imeoer** [comment]
> Can we package the binary in the contrib directory, such as Nydusify, into macos?

@adamqqqplay nydusify and all components in contrib do not support macos yet.

**Comment by adamqqqplay** [comment]
> > Can we package the binary in the contrib directory, such as Nydusify, into macos?
> 
> @adamqqqplay nydusify and all components in contrib do not support macos yet.

@imeoer Ok, I thought it was just not packaged for release

**Comment by killagu** [comment]
https://github.com/cloud-hypervisor/fuse-backend-rs/pull/133

**Review by jiangliu — APPROVED** [review]

**Review by imeoer — APPROVED** [review]

**Pull request merged**